Foundation/FileHandle.swift

```diff
@@ -426,6 +443,13 @@ open class FileHandle : NSObject, NSSecureCoding {
     }
 #endif
 
+    internal convenience init?(fileSystemRepresentation: UnsafePointer<Int8>, flags: Int32, createMode: Int) {
+        let fd = _CFOpenFileWithMode(fileSystemRepresentation, flags, mode_t(createMode))
+        guard fd > 0 else { return nil }
+        
+        self.init(fileDescriptor: fd, closeOnDealloc: true)
+    }
```

Comment:
I think that instead of the FSR typealias, we should do something like this:

```
internal convenience init?(fileSystemRepresentation: UnsafePointer<Int8>, flags: Int32, createMode: Int) {
#if os(Windows)
  var fd: Int -1
  if let path = String(cString: fileSystemRepresentation).withCString(encodedAs: UTF16.self) {
    fd = _CFOpenFileWithMode($0, flags, mode_t(createMode))
  }
#else
  let fd = _CFOpenFileWithMode(fileSystemRepresentation, flags, mode_t(createMode))
#endif
  guard fd > 0 else { return nil }
  self.init(fileDescriptor: fd, closeOnDealloc: true)
}
```

Reply:
I think you mean to use `.cString(using: .utf16)` method?

Reply:
It is done.